lib/connect-backbone-to-react.js

```diff
@@ -128,6 +128,18 @@ module.exports = function connectBackboneToReact(
             (this.context.models && this.context.models[mapKey] === this.models[mapKey])) {
```

Comment:
Also just realized that we're duplicating `this.models[mapKey]` when we have a reference to `model` above. Can we squeeze in a change to reference `model` directly for some code hygiene?